apps/coa/translations/src/en/buttons.json

```diff
@@ -11,5 +11,6 @@
   "proof-of-identity": "Proof of identity",
   "what-you-need": "What you will need",
   "proof-of-address": "Proof of Address",
-  "start-now": "Start now"
+  "start-now": "Start now",
+  "submit-report": "Confirm and submit"
```

Comment:
The button wording should be "Submit change"
I would also suggest to rename `submit-report` key, as could be misleading. Perhaps just `submit` or `submit-change`

Reply:
Updated the button key as "submit" and label should be "Confirm and submit" to matches the figma.